src/wallet/wallet.cpp

```diff
@@ -2173,6 +2172,7 @@ bool CWallet::CommitTransaction(CWalletTx& wtxNew, CReserveKey& reservekey)
             if (!wtxNew.AcceptToMemoryPool(false))
             {
                 // This must not fail. The transaction has already been signed and recorded.
+                // Also the fee never exceeds maxTxFee which forces mempool rejection
```

Comment:
nit: this comment didn't really help me.  maybe `// Wallet uses same maxTxFee as ATMP so tx should not be rejected for fee` but maybe its just unnecessary.